### PR TITLE
Revamp featured datasets

### PR DESCRIPTION
**Pull request by ricardomestre**

### CODE REVIEW

File: ckanext-offgridtheme/ckanext/offgridtheme/static/scripts/maps.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+mapboxgl.accessToken = 'pk.eyJ1IjoiZG1vcmlhcnR5IiwiYSI6Ikd3T29EOWMifQ.-DKJ4ernht84AZmc6Bk51Q';
+var map_1 = new mapboxgl.Map({
+    container: 'ssa-aicd-electricity-map',
+    style: '/data/map-style.json',
```
Comment (olafveerman) [inline ckanext-offgridtheme/ckanext/offgridtheme/static/scripts/maps.js]: Can we use Mapbox's `light-v9` for this? It's being used in the other map previews.

File: ckanext-offgridtheme/ckanext/offgridtheme/static/scripts/maps.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    zoom: 1.5
+});
+
+map_1.on('load', function () {
+  map_1.addLayer({
+    "id": 'data-layer-kano',
+    "type": 'line',
+    "source": {
+      "type": "geojson",
+      "data": "/data/aicd-eletricity-clean.geojson"
```
Comment (olafveerman) [inline ckanext-offgridtheme/ckanext/offgridtheme/static/scripts/maps.js]: We should use the Vector Tiles that we're already serving from S3 for both these datasets.
You can find the tilejson on the metadata of each dataset.
You can check https://github.com/developmentseed/ckanext-mvt/blob/develop/ckanext/mvt/templates/mvtview.html  on how we're including the tilejson in the other previews.

File: ckanext-offgridtheme/ckanext/offgridtheme/templates/home/index.html [hunk 3]
```diff
@@ -20,6 +20,106 @@ <h1 class="home-title">Open data and analytics for a sustainable energy future.<
...
+
+              <li class="card-wrapper dataset-wrapper">
+                <article class="card card-dataset">
+                  <div class="card-inner">
+                    <div class="card-media">
+                      <div class="card-media-map">
+
+                        <div id="kano-electricity-distribution-map" class="data-map"></div>
+
+                        <script src="/scripts/maps.js"></script>
```
Comment (olafveerman) [inline ckanext-offgridtheme/ckanext/offgridtheme/templates/home/index.html]: include this elsewhere

File: ckanext-offgridtheme/ckanext/offgridtheme/static/data/map-style.json [hunk 4]
```diff
@@ -0,0 +1,7781 @@
...
+    "zoom": 6.1940388233735195,
+    "bearing": 0,
+    "pitch": 0,
+    "sources": {
+        "composite": {
+            "url": "mapbox://mapbox.mapbox-terrain-v2,mapbox.mapbox-streets-v7",
+            "type": "vector"
+        }
+    },
+    "sprite": "mapbox://sprites/dmoriarty/ciyp3fhho00062rmng8oc28bl",
```
Comment (olafveerman) [inline ckanext-offgridtheme/ckanext/offgridtheme/static/data/map-style.json]: The console complains about this and the next line

**Comment by olafveerman** [comment]
@DylanMoriarty Overall comments:

- for each of these datasets, we have already generated Vector Tiles which we are serving from S3. We should load the data from there.
- let's use `light-v9` as the base-layer to keep consistency

**Comment by DylanMoriarty** [comment]
Getting a few errors here, but it seems to work. The 'ERROR: Forbidden' ones are also seen on the data view pages. The Source one's seem to be MB complaining about the way I wrote up the sources.

**Comment by olafveerman** [comment]
@DylanMoriarty Made some changes to the code, reducing complexity and adding a check whether source  `data` is loaded. Can you have a look and see if it works for you?

**Comment by DylanMoriarty** [comment]
Works fine for me :+1: @olafveerman

**Pull request merged**